packages/calypso-e2e/src/lib/utils/get-test-account.ts

```diff
@@ -0,0 +1,70 @@
+type Env = 'edge' | 'production';
+
+type SiteType = 'simple' | 'atomic';
+
+type DeepPartial< T > = T extends object
+	? {
+			[ P in keyof T ]?: DeepPartial< T[ P ] >;
+	  }
+	: T;
+
+type Plugin = 'gutenberg' | 'coblocks';
+
+type Accounts = { [ key in SiteType ]: { [ key in Env ]: string } };
+type AccountsIndex = { [ key in Plugin ]: Accounts };
+
+type OverridenAccounts = DeepPartial< Accounts >;
+
+type Options = {
+	isEdge?: boolean;
+	isAtomic?: boolean;
+	overridenAccounts?: OverridenAccounts;
+};
+
+const defaultGutenbergAccounts: Accounts = {
+	simple: { edge: 'gutenbergSimpleSiteEdgeUser', production: 'gutenbergSimpleSiteUser' },
+	atomic: { edge: 'gutenbergAtomicSiteEdgeUser', production: 'eCommerceUser' },
+};
+
+const defaultCoblocksAccounts: Accounts = {
+	simple: { edge: 'coBlocksSimpleSiteEdgeUser', production: 'gutenbergSimpleSiteUser' },
+	// not being used, we don't yet test coblocks edge on AT atm
+	atomic: { edge: 'gutenbergAtomicSiteEdgeUser', production: 'eCommerceUser' },
+};
+
+const accountsIndex: AccountsIndex = {
+	gutenberg: defaultGutenbergAccounts,
+	coblocks: defaultCoblocksAccounts,
+};
+
+function getTestSite(
+	forPlugin: Plugin,
+	options: Options = {
+		isEdge: false,
+		isAtomic: false,
```

Comment:
Style suggestion: could we drop the `is`, so it reads simply `edge`: true/false, `atomic`: true/false?